ash/src/lib.rs

```diff
@@ -33,6 +33,12 @@
 //! Load the Vulkan library linked at compile time using [`Entry::linked()`], or load it at runtime
 //! using [`Entry::load()`], which uses `libloading`. If you want to perform entry point loading
 //! yourself, call [`Entry::from_static_fn()`].
+//!
+//! ## Create features
+//!
+//! * **linked** - Link the Vulkan loader at compile time.
```

Comment:
Shall we list the `(default)` features first? Perhaps put them under separate headings, like how clap does it:

https://docs.rs/clap/latest/clap/#feature-flags

Also, I think I prefer a colon (`:`) better than a hyphen (`-`) here.

Reply:
Sounds good. Wouldn't want a separate heading though because unlike clap there's only 3 features.